public/components/trace_analytics/components/services/services_content.tsx

```diff
@@ -123,29 +123,18 @@ export function ServicesContent(props: ServicesProps) {
       (must: any) => must?.term?.serviceName == null
     );
 
-    if (isServiceTrendEnabled) {
```

Comment:
Should these still be nested under isServiceTrendEnabled?

Reply:
That condition is handled here: https://github.com/opensearch-project/dashboards-observability/pull/2325/files#diff-6c41c2b6f0e21b1b29ad6d3cf28e92d0c5fc6bad2fbead527862b0634246ced8R134-R136